build/crd-samples/operations/imageprepulljob.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: operations.kubeedge.io/v1alpha1
+kind: ImagePrepullJob
+metadata:
+  name: imageprepull
+  labels:
+    description: ImagePrepullLabel
+spec:
+  imagePrepullTemplate:
+    images:
+      - busybox:latest
+    nodes:
+      - kubeedge-dev-linux-baoyue
```

A small question, does this file exist as a user example? Would it be better to add some comments like:
```suggestion
    nodes:
      - kubeedge-dev-linux-baoyue  # Need to replaced with your own node name
```

---

> A small question, does this file exist as a user example? Would it be better to add some comments like:

fixed.